dist/docker/redhat/build_docker.sh

```diff
@@ -72,58 +61,45 @@ bcp() { buildah copy "$container" "$@"; }
 run() { buildah run "$container" "$@"; }
 bconfig() { buildah config "$@" "$container"; }
 
-if [ "$type" == "pro" ]; then
-  container="$(buildah from docker.io/ubuntu:20.04)"
-  bcp dist/docker/pro-attach-config.yaml /pro-attach-config.yaml
-elif [ "$type" == "ubuntu" ]; then
-  container="$(buildah from docker.io/ubuntu:24.04)"
-else
-  echo "$type is not a valid option. supported values: ubuntu|pro"
-  exit 1
-fi
+container="$(buildah from docker.io/redhat/ubi9-minimal:latest)"
 
 packages=(
-    "build/dist/$config/debian/${product}_$version-$release-1_$arch.deb"
-    "build/dist/$config/debian/$product-server_$version-$release-1_$arch.deb"
-    "build/dist/$config/debian/$product-conf_$version-$release-1_$arch.deb"
-    "build/dist/$config/debian/$product-kernel-conf_$version-$release-1_$arch.deb"
-    "build/dist/$config/debian/$product-node-exporter_$version-$release-1_$arch.deb"
-    "tools/cqlsh/build/debian/$product-cqlsh_$version-$release-1_$arch.deb"
-    "tools/python3/build/debian/$product-python3_$version-$release-1_$arch.deb"
+    "build/dist/$config/redhat/RPMS/$arch/${product}-$version-$release.$arch.rpm"
+    "build/dist/$config/redhat/RPMS/$arch/$product-server-$version-$release.$arch.rpm"
+    "build/dist/$config/redhat/RPMS/$arch/$product-conf-$version-$release.$arch.rpm"
+    "build/dist/$config/redhat/RPMS/$arch/$product-kernel-conf-$version-$release.$arch.rpm"
+    "build/dist/$config/redhat/RPMS/$arch/$product-node-exporter-$version-$release.$arch.rpm"
```

Comment:
Why do we still have node exporter here?

Reply:
Dropped, since it's not needed anymore